db/dao/sector_dao.py

```diff
@@ -16,6 +17,50 @@ def create_sector(sector, commit=True):
 
         return sector_db
 
+    @staticmethod
+    def get_total_sectors_count():
+        """ Returns the total sectors created in the
+        daatabase at the moment.
+        Used to increment order.
+        """
+        return DB().session.query(Sector).count()
+
+    @staticmethod
+    def create_sectors_division(sector_size, total_size):
+        """Creates all the possible sector records in the
+        database
+        :param sector_size: Specifies the Sector size
+        :param total_size: Specifies the total disk size
+        """
+        total_sectors = sector_size // total_size
+        for sector in range(total_sectors):
+            SectorDao.create_sector(Sector(
+                order=SectorDao.get_total_sectors_count()
+            ))
+
+    @staticmethod
+    def get_first_unused_sector():
+        """ Returns the first available sector
+        """
+        return DB().session.query(Sector).filter_by(
+            is_used=False
+        ).first()
+
+    @staticmethod
+    def insert_sector_data(sector, data, is_used, file_id):
+        """ Inserts the values in an already created sector
+        record
+        :param sector: Sector object model to be manipulated
+        :param data: Specifies the data to be inserted
+        :param is_used: Specifies whether sector is empty
+        :param file_id: Specifies the id of the file linked
+        with this sector
+        """
+        sector.data = data
```

Comment:
A better method is manipulating `sector` where the function is called. So:

`SectorDao.insert_sector_data(Sector(
   data=data,
   is_used=is_used,
   file_id=file_id
))`